config/v1/0000_10_config-operator_01_infrastructure.crd.yaml-patch

```diff
@@ -0,0 +1,24 @@
+- op: add
+  path: /spec/versions/name=v1/schema/openAPIV3Schema/properties/spec/properties/platformSpec/properties/vsphere/properties/vcenters/items/properties/server/anyOf
+  value:
+    - format: ipv4
+    - format: ipv6
+    - format: hostname
+- op: add
+  path: /spec/versions/name=v1/schema/openAPIV3Schema/properties/spec/properties/platformSpec/properties/vsphere/properties/deploymentZones/items/properties/server/anyOf
+  value:
+    - format: ipv4
+    - format: ipv6
+    - format: hostname
+- op: add
+  path: /spec/versions/name=v1/schema/openAPIV3Schema/properties/spec/properties/platformSpec/properties/vsphere/properties/nodeNetworking/properties/external/properties/excludeNetworkSubnetCidr/items/format
+  value: cidr
+- op: add
+  path: /spec/versions/name=v1/schema/openAPIV3Schema/properties/spec/properties/platformSpec/properties/vsphere/properties/nodeNetworking/properties/external/properties/networkSubnetCidr/items/format
+  value: cidr
+- op: add
+  path: /spec/versions/name=v1/schema/openAPIV3Schema/properties/spec/properties/platformSpec/properties/vsphere/properties/nodeNetworking/properties/internal/properties/excludeNetworkSubnetCidr/items/format
+  value: cidr
+- op: add
+  path: /spec/versions/name=v1/schema/openAPIV3Schema/properties/spec/properties/platformSpec/properties/vsphere/properties/nodeNetworking/properties/internal/properties/networkSubnetCidr/items/format
+  value: cidr
```

Comment:
If you only have one format, can this not be `kubebuilder:validation:Format:=cidr`?

Reply:
For some reason kubebuilder adds the format not under items.
Unsure its a bug or a me problem.